src/NUnitFramework/framework/Constraints/EachItemConstraintResult.cs

```diff
@@ -0,0 +1,40 @@
+namespace NUnit.Framework.Constraints
```

Comment:
This file needs to copyright header. Could you just copy it from another file, and change the year to 2018?